site/public/js/forum.js

```diff
@@ -1413,4 +1413,10 @@ function updateSelectedThreadContent(selected_thread_first_post_id){
             window.alert("Something went wrong while trying to fetch content. Please try again.");
         }
     });
-}
\ No newline at end of file
+}
+
+function changeThreadDisplayOptions(option){
+    thread_id = $('#current-thread').val();
+    document.cookie = "forum_thread_display_option=" + option + ";";
+    window.location.replace(buildCourseUrl(['forum', 'threads', thread_id]) + `?threadOption=${option}`);
```

Comment:
Currently the page needs to get refreshed to reorder the posts which can get inconvinient for users. Consider adding your logic to the function `modifyThreadList` in [`forum.js`](https://github.com/Submitty/Submitty/blob/master/site/public/js/forum.js#L536-L595)